types/stripe-v3/index.d.ts

```diff
@@ -622,13 +622,18 @@ declare namespace stripe {
         interface StripeSourcePaymentResponse extends StripePaymentResponse {
             source: Source;
         }
+        
+        interface StripePaymentMethodPaymentResponse extends StripePaymentResponse {
+            paymentMethod: PaymentMethod;
```

Comment:
There is no PaymentMethod interface or class. DId you mean paymentMethod.PaymentMethod?

Reply:
Weird, within Webstorm this seemed to work totally fine.
I tried changing it, lets have a look.